ISISComputingGroup/ibex_utils
# Ticket5639 galil ini parser

For https://github.com/ISISComputingGroup/IBEX/issues/5639

Adds a parser for galil ini files with unit tests.

## To test
1. Run unit tests (`python -m unittest parser_tests.py`)
1. Download a galil settings file from an instrument (e.g. sans2d)
1. Run the script against the settings file
1. Check in the output file that only the limit and offsets are changed, and the new values are correct
